src/hubmap_translator.py

```diff
@@ -1154,7 +1160,51 @@ def delete_and_recreate_indices(self):
             # Log the full stack trace, prepend a line with our message
             logger.exception(msg)
 
+    """
+    Retrieve the organ types from ontology-api
+    
+    Returns
+    -------
+    dict
+        The available organ types in the following format:
+    
+        {
+            "AO": "Aorta",
+            "BD": "Blood",
+            "BL": "Bladder",
+            "BM": "Bone Marrow",
+            "BR": "Brain",
+            "HT": "Heart",
+            ...
+        }
+    """
+    def get_organ_types(self):
+        # Count on good values in the ontology dict from error checking in main.py
+        ontology_api_url = self._ontology_api_dict['ONTOLOGY_API_BASE_URL']
+        ontology_api_app_context = self._ontology_api_dict['ONTOLOGY_API_APP_CONTEXT']
+
+        target_url = f"{ontology_api_url}/organs/by-code?application_context={ontology_api_app_context}"
```

Comment:
This is where double slashes may happen if the config includes a trailing slash.

Reply:
Reformatted line setting `target_url` to use new constants.